zerver/openapi/zulip.yaml

```diff
@@ -5450,6 +5450,86 @@ paths:
                         "result": "success",
                         "msg": "",
                       }
+  /users/{email}:
+    get:
+      operationId: get_user_by_email
+      tags: ["users"]
+      description: |
+        Fetch details for a single user in the organization.
+
+        `GET {{ api_url }}/v1/users/{email}`
+
+        You can also fetch details on [all users in the organization](/api/get-users).
```

Comment:
I extended this sentence to link to the by-user-Id version, and add commentary on the downsides of fetching by email.